### PR TITLE
Feature/triple double letters

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In this pull request, i add file triple_double_letters.py for practice-3, add module triple_double_letters for main.py
#40

### CODE REVIEW

File: python/practices/practice-3/src/path1/triple_double_letters.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    return False
+
+
+def find_triple_double():
+    """
+    Prints words with triple double letters from word list.
+    """
+
+    fin = open('words.txt', encoding='utf-8')
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/triple_double_letters.py]: Please create the helper's function to read the file.

File: python/practices/practice-3/src/path1/triple_double_letters.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    Parameters:
+        - word (str): The input word to check.
+
+    Returns:
+        - bool: True if the word contains three consecutive double letters,
+        False otherwise.
+    """
+
+    i = 0
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/triple_double_letters.py]: Please update the variable name.

File: python/practices/practice-3/src/path1/triple_double_letters.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
+"""
+Check and prints words containing three consecutive double letters
+"""
+
+
+import os
+
+
+def is_triple_double(word):
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/triple_double_letters.py]: Should update function's name

File: python/practices/practice-3/src/path1/triple_double_letters.py (hunk 2 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path1/triple_double_letters.py]: Done

File: python/practices/practice-3/src/path1/triple_double_letters.py (hunk 1 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path1/triple_double_letters.py]: Done

File: python/practices/practice-3/src/path1/triple_double_letters.py (hunk 3 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path1/triple_double_letters.py]: Done

File: python/practices/practice-3/src/path1/triple_double_letters.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+def main():
+    """
+    Prints words containing three consecutive double letters
+    """
+
+    print(os.path.basename(__file__))
+    file_content = read_file_content(word_file)
+
+    if file_content is not None:
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/triple_double_letters.py]: Please review Writing Idiomatic Python 3, if statement section.

Comment (cuongdoan-agilityio) [reply python/practices/practice-3/src/path1/triple_double_letters.py]: Can you think about including this condition in find_triple_double? I think that would be more reasonable.

Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path1/triple_double_letters.py]: Done

File: python/practices/practice-3/src/path1/triple_double_letters.py [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    Prints words with triple double letters from word list.
+
+    Parameters:
+        - file_content (str): The content of the word list.
+
+    Returns:
+        None
+    """
+
+    if file_content:
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/triple_double_letters.py:48]: Please review the Writing Idiomatic Python 3 for If Statements.
Do not nest multiple if statements inside each other.

**Pull request merged**